src/components/product-card/product-card.tsx

```diff
@@ -8,18 +8,13 @@ export type GalleryCardProps = {
     name: string;
     imageUrl?: string;
     className?: string;
+    outOfStock?: boolean;
     price?: products.PriceData;
 } & React.HTMLAttributes<HTMLDivElement>;
 
-export const ProductCard = ({
-    name,
-    imageUrl,
-    className,
-    price,
-    ...divProps
-}: GalleryCardProps) => {
+export const ProductCard = ({ name, imageUrl, className, price, outOfStock, ...divProps }: GalleryCardProps) => {
```

Comment:
@yurii-ve since you updated printWidth, we need to run prettier on every file (but in separate PR). Unfortunately I forgot to mention that in the original PR